lib/shard/shard_test.go

```diff
@@ -43,7 +44,7 @@ func TestBasic(t *testing.T) {
 		RunWorkload(s.URL, facts, t)
 		return true
 	}
-	if err := quick.Check(f, &quick.Config{MaxCount: 5}); err != nil {
+	if err := quick.check(f, &quick.Config{MaxCount: 5}); err != nil {
```

Comment:
True.

Also, I want to rename `check` (to check a test) with `CheckTest`. Might be easier to not mistake this function for a builtin functions.